### PR TITLE
chore: remove unused config file for label prefixes

### PR DESCRIPTION
**Pull request by codebytere**
We don't use Probot's configuration file for anything to do with label prefixing, so we can simplify this code a bit by removing the unnecessary helper method.

cc @MarshallOfSound

### CODE REVIEW

File: src/utils/label-utils.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Context } from 'probot';
+import { PRStatus } from '../enums';
```
Comment (jkleinsc) [inline src/utils/label-utils.ts]: Why was this added?  It doesn't appear to be used.

Comment (codebytere) [reply src/utils/label-utils.ts]: whoops, removed!
